data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -243,6 +239,12 @@ def next_step(this_workflow: Workflow, skip: dict, context=None):
             logger.info("Performing TUMOR_NORMAL_STEP")
             results.append(tumor_normal_step.perform(this_workflow))
 
+        if "DRAGEN_WTS_STEP" in skiplist:
+            logger.info("Skip performing DRAGEN_WTS_STEP")
+        else:
+            logger.info("Performing DRAGEN_WTS_STEP")
+            results.append(dragen_wts_step.perform(this_workflow))
```

Comment:
At the mo, WTS (transcriptome) are still triggering in Batch fashion. See around [1] for example.

So this will break that part a bit. Because, within this if block, T/N will be detached from Batch view and, trigger by Subject across sequence runs basis then on.

WTS is still per library within this Batch/sequence run.

So. We should better keep these different impedance mis-match, separated with their own if/else block. 

Furthermore, since `wts_alignment_qc` are in their own Batch run context, so we have to better check all in this QC Batch run come clean. Only then we can start of triggering next WTS (transcriptome) Batch run. This logic may be better implemented in step module side i.e. part of pre-step in `dragen_wts_step.preform()`.


[1]: https://umccr.slack.com/archives/C8CG6K76W/p1685863962967909

Reply:
> WTS is still per library within this Batch/sequence run.

Ah okay, I think we should switch to the same logic as TN workflow. If a dragen WTS QC workflow is launched per lane we want to wait until all lanes for that library are completed before launching the WTS workflow right?

Reply:
Hence rather than having the Batcher class as input into the prepare_dragen_wts_jobs step we should instead have a List[LabMetadata] object instead?

Reply:
> I think we should switch to the same logic as TN workflow.

Yup. Agree. It seems business logic/requirement is changed for triggering transcriptome workflow.

> If a dragen WTS QC workflow is launched per lane we want to wait until all lanes for that library are completed before launching the WTS workflow right?

Correct. More below.

> Hence rather than having the Batcher class as input into the `prepare_dragen_wts_jobs` step we should instead have a List[LabMetadata] object instead?

Yep. WTS becomes very much similar to T/N situation. But. Instead of pooling across sequencing runs, it has to find its peer _(like "minions", ya know!)_ that might spread across different lanes within this sequencing batch.

So, we can -- 
* Remove the `Batcher -> Batch/BatchRun` mechanism wrap from `dragen_wts_step`
* The input `this_workflow` of its interface `dragen_wts_step.perform(this_workflow)` is no longer `bcl_convert` workflow anymore
* So we loose `FastqListRows` context along with `Batcher` there
* However, `this_workflow` which is now `type_name == wts_alignment_qc` of WTS lib in per lane has still a link to `Workflow.sequence_run_id` attribute
* With that we can still scope it into the same sequencing run context
  * This will then give ability to reconstruct related `LibraryRun` 
  * And/or as well as all the way back to its parent workflow `type_name == bcl_convert`
  * _(...can use one or both of them as find fit)_ ☝️ 
* Then, we can limit the searching space for finding WTS library its other half ("minions") that went into another lanes

Reply:
I've updated this in a552dccc49fd1a5af04542f541f1ba00cf2ecbe0, would you mind having a look? 

If that's all good I'll fix up the tests (assuming I've broken a few with these changes)

Reply:
Looking so far, Alexis. Added few more comments to attend.